From: conradsnicta <conradsnicta@users.noreply.github.com>
Date: Wed, 26 Feb 2025 13:44:19 +0100
Subject: fix include (#419)

---
 tests/de_test.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/de_test.cpp b/tests/de_test.cpp
index a4bb246..c116c69 100644
--- a/tests/de_test.cpp
+++ b/tests/de_test.cpp
@@ -8,7 +8,7 @@
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
 
-#include "../include/ensmallen.hpp"
+#include <ensmallen.hpp>
 #include "catch.hpp"
 #include "test_function_tools.hpp"
 
