From: conradsnicta <conradsnicta@users.noreply.github.com>
Date: Wed, 26 Feb 2025 13:45:24 +0100
Subject: increase tolerance (#420)

---
 tests/demon_sgd_test.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/demon_sgd_test.cpp b/tests/demon_sgd_test.cpp
index b8055dc..69f7c8b 100644
--- a/tests/demon_sgd_test.cpp
+++ b/tests/demon_sgd_test.cpp
@@ -22,7 +22,7 @@ using namespace ens::test;
 TEST_CASE("DemonSGDLogisticRegressionTest", "[DemonSGDTest]")
 {
   DemonSGD optimizer(0.1, 32, 0.9, 1000000, 1e-9, true, true, true);
-  LogisticRegressionFunctionTest(optimizer, 0.003, 0.006, 6);
+  LogisticRegressionFunctionTest(optimizer, 0.006, 0.006, 6);
 }
 
 /**
